tokendito/okta_helpers.py

```diff
@@ -54,10 +54,6 @@ def okta_verify_api_method(mfa_challenge_url, payload, headers=None):
         logging.debug("Received type of response: {}".format(type(response.text)))
         response = response.text
 
-    if "errorCode" in response:
```

Comment:
why do we remove it?, is that because the `response` never contains the error code ?

Reply:
Hi Jady, we have this logic in the user_session_token function